extensions/guacamole-auth-radius/src/main/java/org/apache/guacamole/auth/radius/conf/RadiusAuthenticationProtocol.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.apache.guacamole.auth.radius.conf;
+
+/**
+ * This enum represents supported RADIUS authentication protocols for
+ * the guacamole-auth-radius extension.
+ */
+public enum RadiusAuthenticationProtocol {
+    
+    // Password authentication protocol
+    PAP("pap"),
+    
+    // Challenge-Handshake AUthentication Protocol
+    CHAP("chap"),
+    
+    // Microsoft CHAP version 1
+    MSCHAPv1("mschapv1"),
+    
+    // Microsoft CHAP version 2
+    MSCHAPv2("mschapv2"),
+    
+    // Extensible authentication protocol with MD5 hashing.
+    EAP_MD5("eap-md5"),
+
+    // Extensible authentication protocol with TLS
+    EAP_TLS("eap-tls"),
+
+    // Extensible authentication protocol with Tunneled TLS
+    EAP_TTLS("eap-ttls");
+
+    // Store the string value used in the configuration file.
+    private final String strValue;
+    
+    /**
+     * Create a new RadiusAuthenticationProtocol object having the
+     * given string value.
+     * 
+     * @param strValue
+     *     The string value of the protocol.
```

Comment:
If I'm going to invoke this constructor (defining a new enum value perhaps), "the string value of the protocol" doesn't give me an understanding of what needs to go here. I would need to go through the code using the value to determine what actually is needed.

Is there a better way to describe this?

Reply:
Took a better slice at this documentation.